runner/database.go

```diff
@@ -11,7 +11,6 @@ import (
 	"strings"
 
 	"github.com/multiprocessio/go-json"
-
```

Comment:
Don't delete this line. It's common practice to separate first-party libraries from 3rd party libraries.

Reply:
Right. My auto formatter in vscode keeps putting your go-json library in the 3rd parties so I save without formatting.